### DISCUSSION TITLE
Lora rank vs Lora Alpha

### DISCUSSION THREAD

**Discussion by AdrianAcala** [axolotl-ai-cloud/axolotl#1133] [General]
Hi,

I just got into training lora models recently and so I'm a noob here so please forgive me if I'm mistaken.

I thought it was generally a best practice to have the lora alpha be double what the lora rank is, but in both your [Mistral](https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/examples/mistral/qlora.yml) and [Phi](https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/examples/phi/phi-qlora.yml) examples, the lora alpha is half of what the lora rank is.

**Comment by NanoCode012** [comment]
I think there's conflicting info on what the best hyper parameter selection is for that. The examples are good as a start.
